gravitee-apim-console-webui/src/portal/homepage/homepage.component.ts

```diff
@@ -65,20 +66,26 @@ export class HomepageComponent {
   private readonly destroyRef = inject(DestroyRef);
   private readonly matDialog = inject(MatDialog);
 
-  readonly portalHomepage: WritableSignal<PortalPageWithDetails | null> = signal(null);
+  readonly portalHomepage: WritableSignal<{ navigationItem: PortalNavigationItem; content: PortalPageContent } | null> = signal(null);
+  // TODO: when update of portal navigation item is implemented, read the actual "published" flag from the navigation/page metadata.
+  // For now default to true so the UI has a sane initial state
+  readonly portalHomepagePublished = signal<boolean>(true);
```

Comment:
This doesn't make a lot of sense right now, but it at least links the portalHomepagePublished with the portalHomepage. This allows you to simplify lines 81 and 131 --> It will be automatically calculated by the compute.
```suggestion
  readonly portalHomepagePublished = computed(() => !!this.portalHomepage());
```